chapter_recurrent-neural-networks/bptt.md

```diff
@@ -140,22 +154,26 @@ $$\begin{aligned}
 
 ## Summary
 
-* Back-propagation through time is merely an application of backprop to sequence models with a hidden state.
+* Backpropagation through time is merely an application of backprop to sequence models with a hidden state.
 * Truncation is needed for computational convenience and numerical stability.
-* High powers of matrices can lead top divergent and vanishing eigenvalues. This manifests itself in the form of exploding or vanishing gradients.
-* For efficient computation intermediate values are cached.
+* High powers of matrices can lead to divergent and diminishing eigenvalues, which further results in gradients exploding or vanishing.
+* For efficient computation, intermediate values are cached.
 
 ## Exercises
 
-1. Assume that we have a symmetric matrix $\mathbf{M} \in \mathbb{R}^{n \times n}$ with eigenvalues $\lambda_i$. Without loss of generality assume that they are ordered in ascending order $\lambda_i \leq \lambda_{i+1}$. Show that $\mathbf{M}^k$ has eigenvalues $\lambda_i^k$.
-1. Prove that for a random vector $\mathbf{x} \in \mathbb{R}^n$ with high probability $\mathbf{M}^k \mathbf{x}$ will by very much aligned with the largest eigenvector $\mathbf{v}_n$ of $\mathbf{M}$. Formalize this statement.
+1. Assume that we have a symmetric matrix $\mathbf{M} \in \mathbb{R}^{n \times n}$ with eigenvalues $\lambda_i$. Without loss of generality, assume that they are ordered in an ascending order $\lambda_i \leq \lambda_{i+1}$. Show that $\mathbf{M}^k$ has eigenvalues $\lambda_i^k$.
+1. Prove that for a random vector $\mathbf{x} \in \mathbb{R}^n$ with high probability $\mathbf{M}^k$, $\mathbf{x}$ will be very much likely aligned with the largest eigenvector $\mathbf{v}_n$ of $\mathbf{M}$. Formalize this statement.
 1. What does the above result mean for gradients in a recurrent neural network?
-1. Besides gradient clipping, can you think of any other methods to cope with gradient explosion in recurrent neural networks?
+1. Besides gradient clipping, can you think of any other methods to cope with gradient explosion of recurrent neural networks?
 
 ## Scan the QR Code to [Discuss](https://discuss.mxnet.io/t/2366)
 
 ![](../img/qr_bptt.svg)
+
+Need to review:
```

Comment:
What's this?

Reply:
The math that we need to discuss about when you are free.

Reply:
Then this section is not ready for merging. Can you add [WIP] in the title?